HedgeModManager/Resources/Codesv2/SonicOrigins.hmm

```diff
@@ -147,3 +147,22 @@ if (*(long*)rsdkGlobals != 0)
         WriteProtected<byte>(*(long*)rsdkGlobals + rsdkGOffset + 0x11C, 0x00);
     }
 }
+
+Code "Always Disable Super Music in S3&K" in "Audio" by "MegAmi" does "Due to how the cheat works, the Super music may still play in some instances, such as after the AIZ Act 1 bombing cutscene."
```

Comment:
Rename this code to "Disable Super Music in S3&K", the "Always" is redundant here.